HeimdallrTests/Core/HeimdallrSpec.swift

```diff
@@ -544,6 +662,8 @@ class HeimdallrSpec: QuickSpec {
                         }, withStubResponse: { request in
                             return OHHTTPStubsResponse(data: NSData(contentsOfFile: self.bundle.pathForResource("request-valid", ofType: "json")!)!, statusCode: 200, headers: [ "Content-Type": "application/json" ])
                         })
+                        
+                        accessTokenParser.reset()
```

Comment:
Yeah, I think the current way doesn't really verify that it is only called once for the initial request. I think a counter would work here. We could also think about introducing [Dobby](https://github.com/rheinfabrik/Dobby) here for recording the invocations. What do you think @hffmnn?